qubole-aws-terraform-deployment/qubole-deployment/modules/hive_metastore/variables.tf

```diff
@@ -0,0 +1,60 @@
+variable "deployment_suffix" {
+}
+
+variable "data_lake_project_region" {
+  default = "ap-southeast-1"
```

Comment:
Variables values are provided in this file as well.

Reply:
yes. variables.tf will be the goto place for configuration.